WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
Agreed to your point. However `if` condition also ensures here that is `className` is undefined, let's not call `addClassName` at all. Otherwise if we remove `if(className) {}`, then every time it will stack into `addClassName => Dwt.addClass => Dwt.delClass` calls, even if `className` is undefined and will return from `Dwt.delClass` method. With this `if`,  we are directly avoiding this when it's not needed at all.

Let me know your thought.

Thanks